src/style-spec/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.22.0-dev",
+  "version": "13.23.0-dev",
```

Comment:
Just a sanity check: does 2.5.0 require a style-spec release?

Reply:
I believe that the [protected field](https://github.com/mapbox/mapbox-gl-js/pull/10968) added to support Studio warrants a new release.